Posh-ACME/Public/Submit-Renewal.ps1

```diff
@@ -9,7 +9,8 @@ function Submit-Renewal {
         [switch]$AllAccounts,
         [switch]$NewKey,
         [switch]$Force,
-        [switch]$NoSkipManualDns
+        [switch]$NoSkipManualDns,
+        [hashtable]$PluginArgs

```

Comment:
Can you add an associated `.PARAMETER` help item down around line 170 with text similar to the one in `New-PACertificate`? Something like this:

> A hashtable containing an updated set of plugin arguments to use with the renewal. So if a plugin has a -MyText string and -MyNumber integer parameter, you could specify them as @{MyText='text';MyNumber=1234}.

Reply:
Done - see 7805fee